css/logo-nav.css

```diff
@@ -289,23 +453,24 @@ a {
     -webkit-box-sizing: border-box;
     box-sizing: border-box;
 }
+
+.companies .col img{
+    display: block;
+    width: 100%
+}
 @media(max-width:1200px) {
     .companies li{
         width:23%;
     }
 }
-@media(max-width:767px) {
+@media(max-width:768px) {
```

Comment:
Was that on purpose ?  Watch out there are some CSS media rules that are inclusive, other that are exclusive, you seem now to have 2 transition at 767 and 768 pixels (it might be a bug in bootstrap)